corehq/apps/accounting/interface.py

```diff
@@ -779,14 +786,9 @@ def rows(self):
                     "data-target": adjust_href,
                     "class": "btn",
                 }
-                columns.extend([
+                columns.append(
                     mark_safe(make_anchor_tag(adjust_href, adjust_name, adjust_attrs)),
-                    mark_safe(make_anchor_tag(
```

Comment:
The "Invoice" column includes a link to the same page by clicking on the invoice id hyperlink.

Example from prod:
<img width="62" alt="screen shot 2015-11-30 at 11 43 09 am" src="https://cloud.githubusercontent.com/assets/1757035/11477785/9c09b812-9757-11e5-9141-89bb6d6ae889.png">

I like this over the ugly button that you only see when you scroll off the screen